overlay.nix

```diff
@@ -135,7 +135,17 @@ let
 
         cpcm-x = callPackage ./pkgs/lib/cpcm-x { };
 
-        crest = callPackage ./pkgs/apps/crest { };
+        crest = callPackage ./pkgs/apps/crest {
+          tblite = super.tblite.overrideAttrs (old: {
```

Comment:
I think that is ok for now. Maybe add a short comment here about the purpose and when this can be removed?